<launch>
    <arg name="image_topic" default="image"/>
    <arg name="velo_topic" default="pcl"/>
    <arg name="odom_topic" default="odom"/>
    <arg name="target_topic" default="/targets/position"/>
    <arg name="engine_path" default="$(find person_detector)/yolov5s.engine"/>

    <group ns="/car1">
        <param name="image_topic" value="$(arg image_topic)"/>
        <param name="velo_topic" value="$(arg velo_topic)"/>
        <param name="odom_topic" value="$(arg odom_topic)"/>
        <param name="target_topic" value="$(arg target_topic)"/>
        <param name="engine_path" value="$(arg engine_path)"/>
        <param name="verbose" value="True"/>

        <node pkg="person_detector" type="detector_node" name="detector_node" output="screen"/>
    </group>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find person_detector)/launch/detection.rviz"/>
</launch>